WowPacketParserModule.V2_5_1_38707/Parsers/CombatLogHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+
+namespace WowPacketParserModule.V2_5_1_38835.Parsers
+{
+    public static class CombatLogHandler
+    {
+        [Parser(Opcode.SMSG_SPELL_NON_MELEE_DAMAGE_LOG)]
+        public static void HandleSpellNonMeleeDmgLog(Packet packet)
```

Comment:
This is the same as `V8_0_1_27101.Parsers.HandleSpellNonMeleeDmgLog`

Reply:
You are right.